### DISCUSSION TITLE
Why are the line breaks not aligned in brackets in Google Colab?

### DISCUSSION THREAD

**Discussion by a7y7a** [mrdbourke/pytorch-deep-learning#1095] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/d5e17db2-33a7-4d5c-b7a1-8bbcf101556c)
Hello, I press the Enter key in the brackets in Google Colab to wrap the line, but the next line does not align with the previous line like in the video

**Comment by LuluW8071** [comment, marked as answer]
@a7y7a Ensure you have these settings under tools>settings>editor
![image](https://github.com/user-attachments/assets/747a5436-3c57-4d88-96df-39aff128bc63)

**Reply by a7y7a** [reply]
Sorry, I have these settings but it still don't work
![image](https://github.com/user-attachments/assets/e0f4e95e-fa70-4fcf-91e4-809e6a2ee736)
![image](https://github.com/user-attachments/assets/008a55bf-0465-4277-96d7-5fffc97b163d)

**Reply by LuluW8071** [reply]
I could only replicate that error when a code snippet gets copy pasted. Are you sure indentation happens when u r writing the code.

**Reply by a7y7a** [reply]
> I could only replicate that error when a code snippet gets copy pasted. Are you sure indentation happens when u r writing the code.

The code is as follows.
https://colab.research.google.com/drive/1CmCV9IhA5Fdi0bOcohAj5QJ_S8_VM0wp#scrollTo=tDIzPhudfWnt&line=1&uniqifier=1

**Reply by a7y7a** [reply]
@LuluW8071 I found it, it's a problem with the monospaced font in Microsoft Edge! Thank you!
